tools/ci_build/github/windows/helpers.ps1

```diff
@@ -635,16 +635,18 @@ function Install-ONNX {
     if ($lastExitCode -ne 0) {
       exit $lastExitCode
     }
-
+    $temp_dir = Get-TempDirectory
+    $new_requirements_text_file = Join-Path $temp_dir "new_requirements.txt"
     Write-Host "Installing python packages..."
-    [string[]]$pip_args = "-m", "pip", "install", "-qq", "--disable-pip-version-check", "setuptools>=68.2.2", "wheel", "numpy", "protobuf==$protobuf_version"
+    Get-Content "$src_root\tools\ci_build\github\linux\docker\inference\x86_64\python\cpu\scripts\requirements.txt" | Select-String -pattern 'onnx' -notmatch | Out-File $new_requirements_text_file
```

Comment:
Are all the dependencies in this requirements.txt file necessary to install onnx? Would extracting just the numpy version dependency from onnxruntime's requirements.txt file be sufficient instead? For example, this is what is done in the rel1-1.18.1 branch:
```ps1
    # Get the required numpy version from onnxruntime's requirements.txt (e.g., numpy>=1.21.6,<2.0)
    $numpy_req_match_info = Get-Content "$src_root\requirements.txt" | Select-String -pattern 'numpy'
    $numpy_to_install = $numpy_req_match_info.ToString()

    [string[]]$pip_args = "-m", "pip", "install", "-qq", "--disable-pip-version-check", "setuptools>=68.2.2", "wheel", "$numpy_to_install", "protobuf==$protobuf_version"
```

Reply:
The [requirements.txt](https://github.com/microsoft/onnxruntime/blob/main/tools/ci_build/github/linux/docker/inference/x86_64/python/cpu/scripts/requirements.txt) file here it uses contains more packages than the one in ONNX's repo, and package versions are different. 

ONNX's reqiurements.txt has:
```
numpy>=1.20
protobuf>=3.20.2
```
Here I want to replace them to:
```
numpy==2.0.0
protobuf==(the protobuf version that ORT uses)
``` 
The change of the first line is the reason why I changed the requirements.txt file. Also we need to ensure setuptools and wheel are also correctly installed. So I think it's either to just reuse the same requirements.txt file that our Linux build pipelines are using.

Reply:
The old code has a problem that the packages specified here in this powershell script cannot be scanned by  component governance security scanner.  So we don't know if any of the packages is out of date. We should put all such python deps in requirements.txt files.